linux/mcompile/keymap/meson.build

```
@@ -19,9 +19,14 @@ cpp_files = files(
     'mc_import_rules.cpp',
     'u16.cpp',)
 
+comon_include_dir = [
+  include_directories('../../../../common/include'),
```

I think that's one level up to much:

```suggestion
  include_directories('../../../common/include'),
```

---

OJK-done